pages/clips.html

```diff
@@ -0,0 +1,11 @@
+---
+permalink: clips/
+---
+<h1>New Clips</h1>
+
+{% for c in site.data.clips %}
```

Comment:
For now, this'd need to be `site.data.public.clips`, unless you want to hoist `_data/public/clips.yml` up to `_data/clips.yml` in this here PR.